vispy/app/backends/_qt.py

```diff
@@ -230,6 +230,20 @@ def _vispy_get_native_app(self):
         # Return
         return app
 
+    def _vispy_sleep(self, duration_sec):
+        if self._vispy_get_backend_name() == "PyQt4":
+            import PyQt4.QtTest
+            PyQt4.QtTest.QTest.qWait(duration_sec * 1000)  # in ms
+
+        elif self._vispy_get_backend_name() == "PyQt5":
+            import PyQt5.QtTest
+            PyQt5.QtTest.QTest.qWait(duration_sec * 1000)  # in ms
```

Comment:
You shouldn't need to do this, instead import in the individual backends `_pyqt4.py`, `_pyside.py` etc. and then it should be available here. Triaging has already been done in those files, so it is more DRY and orthogonal not to do it here